cli/stop_file_writer.py

```diff
@@ -0,0 +1,132 @@
+import argparse
+import sys
+import time
+from datetime import datetime, timedelta
+from time import time as current_time
+
+from cli.start_file_writer import is_empty
+from file_writer_control import JobHandler, JobState, WorkerCommandChannel
+
+
+def cli_parser() -> argparse.Namespace:
+    """
+    Parser for the command line interface.
+    """
+
+    fw_parser = argparse.ArgumentParser(
+        fromfile_prefix_chars="@", description="FileWriter Stopper"
+    )
+
+    fw_parser.add_argument(
```

Comment:
I presume -s and --sa both expect a job id to stop the correct FW job? Would it be possible to add a cli command here to list available job id's that are subscribing to the command topic?

Reply:
good idea. But may require looking for all start messages on the topic. or may be there is an easy way.